app/controllers/ExpressionController.java

```diff
@@ -177,4 +229,5 @@ private ExpressionController(final int maxLimit, final ExpressionRepository expr
 
     private static final int DEFAULT_MAX_LIMIT = 1000;
     private static final Logger LOGGER = LoggerFactory.getLogger(ExpressionController.class);
+    private static final ObjectMapper OBJECT_MAPPER = new ObjectMapper();
```

Comment:
You should probably add the build to the OM here too given if registers the referenced types recursively and so would be future-proofed against any changes to the view.